### DISCUSSION TITLE
Case Templates no longer displayed in Reflex-UI

### DISCUSSION THREAD

**Discussion by ar3diu** [reflexsoar/reflex-api#228] [Q&A] [answered]
Hi,

We encountered a strange issue today when Case Templates suddenly stopped being displayed in the UI. Though, they seem to be loaded from Elastic according to browser network tools: 

![image](https://user-images.githubusercontent.com/61625853/179933478-b9395046-7538-410d-b165-8502b1de8b02.png)

We can also apply them when creating a case starting from an Event in the Queue.
![image](https://user-images.githubusercontent.com/61625853/179939415-5d95d4e0-1049-45e7-a6a2-55ede56f6388.png)

Tried to switch browsers and to rebuild reflex-api, reflex-ui and memcached docker containers without success. 
Also, searched through the docker logs but couldn't find anything related. :man_shrugging: 

Any suggestions on how I can troubleshoot further?

**Comment by enceladus3** [comment]
Issue occurred right after creating a new Case Template without Tasks.

**Comment by ar3diu** [comment, marked as answer]
So the last Case Template was created by a user with the default analyst role. Right after that, all Case Templates stopped being displayed.
Fixed it by removing the the elasticsearch _doc with the last Case Template created.
